Editor/AGS.Editor/Resources/agsdefns.sh

```diff
@@ -2143,8 +2143,14 @@ builtin managed struct Object {
   import function RunInteraction(CursorMode);
   /// Instantly moves the object to have its bottom-left at the new co-ordinates.
   import function SetPosition(int x, int y);
+#ifdef SCRIPT_API_v3507
+  /// Sets the object to use the specified view, ahead of doing an animation.
+  import function SetView(int view, int loop=0, int frame=0);

```

Comment:
Wouldn't changing the defaults here be enough for newer code while keeping backward compatibility with older games?

Reply:
Strictly speaking it won't be enough if you import existing scripts that rely on old behavior.